Install of realm-js in Raspberry PI 4 fails.

I have tried to install the realm-js drivers on a raspberry Pi Model 4. The installation fails. I amusing Ubuntu 20, 64 bit

`Linux ubuntu 5.4.0-1015-raspi #15-Ubuntu SMP Fri Jul 10 05:34:24 UTC 2020 aarch64 aarch64 aarch64 GNU/Linux`

`ubuntu@ubuntu:~/sandbox/HomeAutomationRPI$ {color:#FF0000}npm install realm@beta{color`}
`npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142`
`npm WARN deprecated core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.`
`npm WARN deprecated har-validator@5.1.5: this library is no longer supported``> realm@10.0.0-beta.13 install /home/ubuntu/sandbox/HomeAutomationRPI/node_modules/realm`
`> node-pre-gyp install --fallback-to-build -j 8 ${npm_config_arch:+--target_arch=${npm_config_arch`}}`internal/modules/cjs/loader.js:638`
{{ throw err;}}
{{ ^}}`Error: Cannot find module '../'`
{{ at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)}}
{{ at Function.Module._load (internal/modules/cjs/loader.js:562:25)}}
{{ at Module.require (internal/modules/cjs/loader.js:692:17)}}
{{ at require (internal/modules/cjs/helpers.js:25:18)}}
{{ at Object.<anonymous> (/home/ubuntu/sandbox/HomeAutomationRPI/node_modules/.bin/node-pre-gyp:15:20)}}
{{ at Module._compile (internal/modules/cjs/loader.js:778:30)}}
{{ at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)}}
{{ at Module.load (internal/modules/cjs/loader.js:653:32)}}
{{ at tryModuleLoad (internal/modules/cjs/loader.js:593:12)}}
{{ at Function.Module._load (internal/modules/cjs/loader.js:585:3)}}
`npm WARN deprecated-react-native-listview@0.0.6 requires a peer of react@* but none is installed. You must install peer dependencies yourself.`
`npm WARN deprecated-react-native-listview@0.0.6 requires a peer of react-native@* but none is installed. You must install peer dependencies yourself.`
`npm WARN garden-node@1.0.0 No repository field.``npm ERR! code ELIFECYCLE`
`npm ERR! errno 1`
`npm ERR! realm@10.0.0-beta.13 install: `node-pre-gyp install --fallback-to-build -j 8 ${npm_config_arch:+--target_arch=${npm_config_arch``}}
`npm ERR! Exit status 1`
{{npm ERR! }}
`npm ERR! Failed at the realm@10.0.0-beta.13 install script.`
`npm ERR! This is probably not a problem with npm. There is likely additional logging output above.``npm ERR! A complete log of this run can be found in:`
`npm ERR! /home/ubuntu/.npm/_logs/2020-09-28T15_49_08_231Z-debug.log`

ubuntu@ubuntu:~/sandbox/HomeAutomationRPI$【 highlighted #FF0000 】npm install realm@10.0.0-beta.12【 end highlighted 】
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated core-js@1.2.7: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported

> realm@10.0.0-beta.12 install /home/ubuntu/sandbox/HomeAutomationRPI/node_modules/realm
> node-pre-gyp install --fallback-to-build -j 8

internal/modules/cjs/loader.js:638
 throw err;
 ^

Error: Cannot find module '../'
 at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
 at Function.Module._load (internal/modules/cjs/loader.js:562:25)
 at Module.require (internal/modules/cjs/loader.js:692:17)
 at require (internal/modules/cjs/helpers.js:25:18)
 at Object.<anonymous> (/home/ubuntu/sandbox/HomeAutomationRPI/node_modules/.bin/node-pre-gyp:15:20)
 at Module._compile (internal/modules/cjs/loader.js:778:30)
 at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
 at Module.load (internal/modules/cjs/loader.js:653:32)
 at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
 at Function.Module._load (internal/modules/cjs/loader.js:585:3)
npm WARN deprecated-react-native-listview@0.0.6 requires a peer of react@* but none is installed. You must install peer dependencies yourself.
npm WARN deprecated-react-native-listview@0.0.6 requires a peer of react-native@* but none is installed. You must install peer dependencies yourself.
npm WARN garden-node@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! realm@10.0.0-beta.12 install: `node-pre-gyp install --fallback-to-build -j 8`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the realm@10.0.0-beta.12 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR! /home/ubuntu/.npm/_logs/2020-09-28T16_36_13_295Z-debug.log

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -21,6 +21,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Internal
 * Changed CI to abort if changes are exclusively made to the /packages directory. ([#3298](https://github.com/realm/realm-js/pull/3298)) & ([#3307](https://github.com/realm/realm-js/pull/3307))
+* Publish binaries for Raspberry Pi. ([#3272](https://github.com/realm/realm-js/issues/3272), since v10.0.0-beta.13)
 
 10.0.0-rc.1 Release notes (2020-10-1)
 =============================================================

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -488,6 +488,7 @@ def publish(nodeVersions, electronVersions, dependencies, tag) {
     for (def platform in ['macos', 'linux', 'windows-ia32', 'windows-x64']) {
       unstash "pre-gyp-${platform}-${nodePublishVersion}"
     }
+    unstash "pre-gyp-linux-armhf-${nodePublishVersion}"
 
     withCredentials([[$class: 'FileBinding', credentialsId: 'c0cc8f9e-c3f1-4e22-b22f-6568392e26ae', variable: 's3cfg_config_file']]) {
       sh "s3cmd -c \$s3cfg_config_file put --multipart-chunk-size-mb 5 realm-* 's3://static.realm.io/node-pre-gyp/napi-v${dependencies.NAPI_VERSION}/realm-v${dependencies.VERSION}/'"

